ansible_community_steering_committee.rst

```diff
@@ -0,0 +1,110 @@
+5.  Andrew Klychkov (andersson007_)
+6.  Alicia Cozine (acozine)
+7.  Sorin Sbarnea (zbr)
+8.  Jill Rouleau (jillr)
+9. Brad Thornton (cidrblock)
+10. Dylan Silva (thaumos)
+
+*Key: Full name (IRC Nickname)*
+
+-  Currently, all the Steering Committee members are selected based on
+   their active contribution to the Ansible Project.
+-  In future, the selection process will be based on the election model
+   through community.
+
+New Policy/Proposals & Inclusion Requests:
+==========================================
+
+Requests regarding new collection inclusion requests should be submitted
+to this `GitHub
+discussion <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`__
+and requests regarding new policy/proposals should be submitted through
+a new pull request to
+`ansible-collections/overview <https://github.com/ansible-collections/overview>`__
+GitHub repo.
+
+Also, we earlier used a `Community Working Group Meeting
+Agenda <https://github.com/ansible/community/issues/539>`__ to track
+agenda topics that needed to be discussed in the weekly Ansible
+Community IRC meetings. Now, to categorize each topic separately we’ve
+created a new GitHub repo
+“`community-topics <https://github.com/ansible-community/community-topics>`__”.
+If you want to discuss an idea, improvements or submitted new
+Policy/Proposals & New Collection Inclusion Requests then create a `new
+issue <https://github.com/ansible-community/community-topics/issues/new>`__
+in the `repo <https://github.com/ansible-community/community-topics>`__
+as a topic, and it’ll be discussed publicly in the weekly IRC meeting.
+After every meeting, the meeting minutes/summaries and meeting logs will
+be posted to the `original
+issue <https://github.com/ansible/community/issues/539>`__.
+
+If you cannot join the meeting or from another time zone then you can
+directly use GitHub Discussions feature to discuss your new
+proposals/collection inclusion requests and the steering committee will
+discuss the final conclusions during the weekly IRC meeting and then
+will vote on them.
+
+Meetings:
+=========
+
+Ansible Community meeting happens on a weekly basis on
+#ansible-community IRC channel every Wednesday `at 18:00
+UTC <https://raw.githubusercontent.com/ansible/community/main/meetings/ical/community.ics>`__.
+
+Agenda
+~~~~~~
+
+You can find the Ansible Community Meeting agenda at `Community Working
+Group Meeting
+Agenda. <https://github.com/ansible/community/issues/539>`__
+
+Minutes of Meeting
+~~~~~~~~~~~~~~~~~~
+
+Ansible Community weekly meeting minutes can be found at the `fedora
+meetbot
+site <https://meetbot.fedoraproject.org/sresults/?group_id=ansible-community&type=channel>`__
+and the same will be posted to `Ansible Devel Mailing
+List <https://groups.google.com/g/ansible-devel>`__ after every meeting.
+
+Mailing List:
+-------------
+
+Any important announcements regarding Ansible Community Project will be
+posted to ``ansible-devel@googlegroups.com``.
```

Comment:
```suggestion
Mailing List:
-------------

Any important announcements regarding Ansible Community Project will be
posted to ``ansible-devel@googlegroups.com``.
```
Is it relevant? If it's relevant for core only, maybe we should exclude the paragraph not to confuse folks. I would put Bullhorn inestead.

Reply:
Initially, we thought of creating a new mailing list for our own but Toshio suggested to use the existing ansible-devel google group as that group also includes folks from the community.

Reply:
Have it worked so far? If not
1. It should be actively used:) i.e. at least, the decisions made on community topics should be delivered via this
2. Or it should be removed because someone can rely on this source and think that nothing happens